airbyte-integrations/connectors/source-cart/acceptance-test-config.yml

```diff
@@ -8,23 +8,21 @@ tests:
     - config_path: "secrets/config.json"
       status: "succeed"
     - config_path: "integration_tests/invalid_config.json"
-      status: "exception"
+      status: "failed"
+      timeout_seconds: 180
   discovery:
     - config_path: "secrets/config.json"
   basic_read:
     - config_path: "secrets/config.json"
       configured_catalog_path: "integration_tests/configured_catalog.json"
       validate_output_from_all_streams: yes
-# TODO uncomment this block to specify that the tests should assert the connector outputs the records provided in the input file a file
-#      expect_records:
-#        path: "integration_tests/expected_records.txt"
-#        extra_fields: no
-#        exact_order: no
-#        extra_records: yes
-  incremental: # TODO if your connector does not implement incremental sync, remove this block
+      timeout_seconds: 1800
```

Comment:
this is a huge time, can we reduce the amount of data that we read?

Reply:
in our live store there are 3 streams that has the data prior to March of this year, these are ` {'orders', 'order_payments', 'products'}`. The most huge customers_cart has the most of data. To read our data completely from the beginning of this year it takes around 20-21 minutes (git test runs time). We may not want to fill the real store with a dummy data, as well don't want to skip the streams that really has the data, or use start_date for each stream etc.. 1800 is 30m with some reserve to pass the test, but average test time is up to 23 min

Reply:
I will check note above seems my comment is incorrect